fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchReferenceTest.java

```diff
@@ -33,18 +34,29 @@
     private String patientResourceIdWithReferenceUrl;
     @SuppressWarnings("unused")
     private String patientWithLiteralReference;
+    @SuppressWarnings("unused")
+    private String patientResourceIdWithPractitionerReferenceTypeId;
+    @SuppressWarnings("unused")
+    private String patientResourceIdWithPractitionerRoleReferenceTypeId;
 
     /*
      * creates the various test cases for a patient with a reference
      */
-    public String createPatientWithReference(String reference) throws Exception {
+    public String createPatientWithReference(String field, String reference) throws Exception {
         WebTarget target = getWebTarget();
 
         // Build a new Patient and then call the 'create' API.
         Patient patient = TestUtil.readLocalResource("Patient_JohnDoe.json");
 
-        patient =
-                patient.toBuilder().gender(AdministrativeGender.MALE).managingOrganization(Reference.builder().display(com.ibm.fhir.model.type.String.of("Test Organization")).reference(com.ibm.fhir.model.type.String.of(reference)).build()).build();
+        if (field.equals("organization")) {
```

Comment:
a cleaner way to write this is 

"organization".equals(field)

Refer to https://stackoverflow.com/a/10911432/1873438

Reply:
Done.